config/main.py

```diff
@@ -1358,16 +1358,16 @@ def naming_mode_alias():
 #
 # 'syslog' group ('config syslog ...')
```

Comment:
@tsvanduyn please align the description

Reply:
The command syntax did not change. Please see my comment below.

Reply:
@jleveque thanks for pointing out.